dev-guide/featuresets.md

```diff
@@ -403,6 +403,23 @@ The biggest difference is that our configuration is one of our APIs.
 Configuration never goes away.
 Feature Gates can and do disappear by design.
 
+## Can you disable a Feature Gate that is enabled?
+This depends on what you mean by the questions.
+If you mean a flow like:
+1. Cluster put into TechPreviewNoUpgrade.  This enables featuregate/X.
+2. Cluster put into Default.  This disables featuregate/Y.
+
+No, we intentionally do not allow transition out of TechPreviewNoUpgrade or DevPreviewNoUpgrade.
+We prevent this using API validation that disallows these transitions.
+
+If you mean a flow like:
+1. 4.18 Default has featuregate/Y enabled by default.
+2. 4.19 Default has the featuregate/Y removed for some reason (this occassionally happens in kube).
+
+Yes.  This is the exception, not the rule, but it does happen on rare occasions.
+
+For downstream feature gates, the team will always know that this has happened.
```

Comment:
Could you be a little more specific? Is this the release team you're referring to?

Reply:
> Could you be a little more specific? Is this the release team you're referring to?

Will update and push.  I mean the team that owns the featuregate, since the state of an OCP owned featuregate won't change without contacting the responsible team.